src/components/VAccordion.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="accordion">
+    <div
+      class="accordion-header"
+      :class="{'accordion-header-active': visible}"
+      tabindex="0"
+      @click="open"
+      @keypress.enter.space="open"
+    >
+      <div class="accordion-header-wrapper">
+        <slot name="header" />
+      </div>
+      <div class="accordion-icon">
+        <simple-arrow-down v-if="active" />
+        <simple-arrow-up v-else />
+      </div>
+    </div>
+    <transition name="accordion">
+      <div
+        v-show="visible"
+        class="accordion-content"
+        :style="'max-height: '+height+'px'"
+      >
+        <slot name="content" />
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import SimpleArrowDown from '../components/icons/SimpleArrowDown'
+import SimpleArrowUp from '../components/icons/SimpleArrowUp'
+
+export default {
+  components: {
+    SimpleArrowDown,
+    SimpleArrowUp
+  },
+  props: {
+    openOnLoad: {
```

Comment:
being nitpicky here but what do you think about standardizing on is/can/has/should type prefixes for boolean prop names. (e.g. `shouldOpenOnLoad`). I like it because it makes it more clear what the  on/off flags are when looking at templates.

Reply:
sure, i like that better! i'll update :)